{"uuid": "d258f6ff-7d69-4c73-8bc4-6aa6dee5adb9", "children": ["458254f8-7ae9-40c7-9d78-8d8c363ba21d"], "befores": [{"name": "launch_browser", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_INTERNET_DISCONNECTED\n  (Session info: chrome=109.0.5414.75)\nStacktrace:\nBacktrace:\n\t(No symbol) [0x01066643]\n\t(No symbol) [0x00FFBE21]\n\t(No symbol) [0x00EFDA9D]\n\t(No symbol) [0x00EF9E22]\n\t(No symbol) [0x00EEFCFD]\n\t(No symbol) [0x00EF1101]\n\t(No symbol) [0x00EEFFDD]\n\t(No symbol) [0x00EEF3BC]\n\t(No symbol) [0x00EEF2D8]\n\t(No symbol) [0x00EEDC68]\n\t(No symbol) [0x00EEE512]\n\t(No symbol) [0x00EFF75B]\n\t(No symbol) [0x00F67727]\n\t(No symbol) [0x00F4FD7C]\n\t(No symbol) [0x00F66B09]\n\t(No symbol) [0x00F4FB76]\n\t(No symbol) [0x00F249C1]\n\t(No symbol) [0x00F25E5D]\n\tGetHandleVerifier [0x012DA142+2497106]\n\tGetHandleVerifier [0x013085D3+2686691]\n\tGetHandleVerifier [0x0130BB9C+2700460]\n\tGetHandleVerifier [0x01113B10+635936]\n\t(No symbol) [0x01004A1F]\n\t(No symbol) [0x0100A418]\n\t(No symbol) [0x0100A505]\n\t(No symbol) [0x0101508B]\n\tBaseThreadInitThunk [0x758DFEF9+25]\n\tRtlGetAppContainerNamedObjectPath [0x77A67BBE+286]\n\tRtlGetAppContainerNamedObjectPath [0x77A67B8E+238]\n\n", "trace": "  File \"C:\\Users\\User\\PycharmProjects\\SportyfriendsProject\\venv\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\User\\PycharmProjects\\SportyfriendsProject\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 1129, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\User\\PycharmProjects\\SportyfriendsProject\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 908, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\Users\\User\\PycharmProjects\\SportyfriendsProject\\tests\\conftest.py\", line 13, in launch_browser\n    driver.get(BASE_URL)\n  File \"C:\\Users\\User\\PycharmProjects\\SportyfriendsProject\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 449, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"C:\\Users\\User\\PycharmProjects\\SportyfriendsProject\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 440, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\User\\PycharmProjects\\SportyfriendsProject\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1678400891458, "stop": 1678400895252}], "start": 1678400891458, "stop": 1678400895659}